pricing.js

```diff
@@ -73,11 +73,36 @@ function calculateProductPrice(product, employee, selectedOptions) {
   }
 }
 
+function calculateCommuterPricePerType(type, costs) {
+  const eeCost = costs.find(cost => {
+    return cost.type === type
+  })
+  return eeCost.price
+}
+
+function calculateCommuterPrice(commute, selectedOptions) {
+
+  switch (selectedOptions.benefit) {
+    case 'train':
+    case 'parking':
+      break
+    default:
+      throw new Error(`Unknown commute type: ${selectedOptions.benefit}`)
+  }
```

Comment:
I like the idea of throwing an error for unknown type but this isn't a good use of a switch, an if would have been better.